src/types.rs

```diff
@@ -142,13 +142,13 @@ impl TypeBuilder for CoreTypeConcrete {
             CoreTypeConcrete::EcState(_) => todo!(),
             CoreTypeConcrete::Felt252(_) => get_integer_layout(252),
             CoreTypeConcrete::GasBuiltin(_) => get_integer_layout(64),
-            CoreTypeConcrete::BuiltinCosts(_) => todo!(),
-            CoreTypeConcrete::Uint8(_) => todo!(),
-            CoreTypeConcrete::Uint16(_) => todo!(),
-            CoreTypeConcrete::Uint32(_) => todo!(),
-            CoreTypeConcrete::Uint64(_) => todo!(),
-            CoreTypeConcrete::Uint128(_) => todo!(),
-            CoreTypeConcrete::Uint128MulGuarantee(_) => todo!(),
+            CoreTypeConcrete::BuiltinCosts(_) => Layout::new::<()>(),
+            CoreTypeConcrete::Uint8(_) => get_integer_layout(8),
+            CoreTypeConcrete::Uint16(_) => get_integer_layout(16),
+            CoreTypeConcrete::Uint32(_) => get_integer_layout(32),
+            CoreTypeConcrete::Uint64(_) => get_integer_layout(64),
+            CoreTypeConcrete::Uint128(_) => get_integer_layout(128),
+            CoreTypeConcrete::Uint128MulGuarantee(_) => Layout::new::<()>(),
```

Comment:
Same here, I think we should match the vm's behavior even though we won't really end up using this type for anything. Or at least have the option to, with a flag or something. What do you think?